src/commcare_cloud/fab/deploy_diff.py

```diff
@@ -90,7 +116,8 @@ def get_diff_context(self):
             "LABELS_TO_EXPAND": LABELS_TO_EXPAND,
             "errors": [],
             "warnings": [],
-            "changelogs": self.get_changelog_context()
+            "changelogs": self.get_changelog_context(),
+            "maintenance_prs": self.get_maintenance_pr_context()
```

Comment:
It would be better to skip this method call entirely if `is_dimagi_env` is True and just return a static value.

Reply:
True